content/en/service_management/workflows/private_actions/use_private_actions.md

```diff
@@ -76,6 +76,20 @@ From the **Private Action Runner** page in [Workflow Automation][6] or [App Buil
    You can safely ignore the error `DATADOG TRACER DIAGNOSTIC - Agent Error: connect ECONNREFUSED`.
 
 [101]: https://docs.docker.com/compose/compose-application-model/
+{{% /tab %}}
+
+{{% tab "Kubernetes" %}}
+1. Click **Kubernetes**.
+1. Confirm that you have installed `kubectl` on your machine by running `kubectl version` and verifying that there is output, then check the box on the **Private Action Runner** page.
+1. Confirm that you have installed `helm` on your machine by running `helm version` and verifying that there is output, then check the box on the **Private Action Runner** page.
+1. Confirm that you have sufficient permissions to create Kubernetes resources in your cluster by running `kubectl get pods` and verifying that the output is not an error, then check the box on the **Private Action Runner** page.<br>Further instructions appear in the app.
+1. Follow the instructions provided in the app to:
+    1. Enroll the runner and generate the config.
+    1. Create a values.yaml file.
+    1. Add the **Private Action Runner** to your Helm repositories.
+    1. Install the Helm chart.
```

Comment:
I think adding something along "run `kubectl get pods -w` and verify that the private action runner pod becomes healthy" might be useful ?

Reply:
Creating and testing a connection would pick up on whether the pod is responding, but it’d be helpful for the customer to double-check that the pod is fully healthy and stable before moving on. The **Ready for use** status doesn’t completely cover pod health, as it’s specific to Kubernetes, so I agree with Gabriel that adding this step would be useful.

Reply:
Good to know, thanks. Will add now